mllib/src/main/scala/org/apache/spark/mllib/linalg/Vectors.scala

```diff
@@ -38,8 +38,6 @@ import org.apache.spark.sql.types._
 
 /**
  * Represents a numeric vector, whose index type is Int and value type is Double.
- *
- * Note: Users should not implement this interface.
```

Comment:
Use your own preferred linear algebra library? I'm not sure this needs an alternative. It's something that's not intended to be subclassed and that's about all of the message.